docs/modules/hdfs/pages/usage-guide/listenerclass.adoc

```diff
@@ -2,6 +2,8 @@
 
 The operator deploys a xref:listener-operator:listener.adoc[Listener] for each DataNode and NameNode pod. They both default to only being accessible from within the Kubernetes cluster, but this can be changed by setting `.spec.{data,name}Nodes.config.listenerClass`.
 
+Please note that JournalNodes are not accessible from outside of the Kubernetes cluster.
```

Comment:
```suggestion
Note that JournalNodes are not accessible from outside the Kubernetes cluster.
```

[use of "please"](https://developers.google.com/style/tone#politeness-and-use-of-please)